Content.Server/Spawners/EntitySystems/ConditionalSpawnerSystem.cs

```diff
@@ -89,24 +89,22 @@ private void Spawn(RandomSpawnerComponent component)
             }
 
             if (component.Chance != 1.0f && !_robustRandom.Prob(component.Chance))
-            {
                 return;
-            }
 
             if (component.Prototypes.Count == 0)
             {
                 Logger.Warning($"Prototype list in RandomSpawnerComponent is empty! Entity: {component.Owner}");
                 return;
             }
 
-            if(!Deleted(component.Owner))
-            {
-                var x_negative = _robustRandom.Prob(0.5f) ? -1 : 1;
-                var y_negative = _robustRandom.Prob(0.5f) ? -1 : 1;
+            if (Deleted(component.Owner)) return;
 
-                var entity = EntityManager.SpawnEntity(_robustRandom.Pick(component.Prototypes), Transform(component.Owner).Coordinates);
-                Transform(entity).LocalPosition += new Vector2(_robustRandom.NextFloat() * component.Offset * x_negative, _robustRandom.NextFloat() * component.Offset * y_negative);
-            }
+            var xOffset = _robustRandom.NextFloat(-1, 1);
+            var yOffset = _robustRandom.NextFloat(-1, 1);
```

Comment:
still need to scale by `component.Offset`

Reply:
Just did NextFloat(-offset, offset)